Sources/PT.PM.Tests/UstPreprocessorTests.cs

```diff
@@ -103,7 +103,7 @@ public void Preprocess_MultiMultiPattern_RemovedDuplicates()
 
             var statements = ((PatternStatements)result).Statements;
             var invocation = (PatternInvocationExpression)statements.ElementAt(0);
-            Assert.AreEqual(1, invocation.Arguments.Args.Count(child => child is PatternMultipleExpressions));
+            Assert.AreEqual(1, (invocation.Arguments as PatternArgs)?.Args.Count(child => child is PatternMultipleExpressions));
```

Comment:
The type of `invocation.Arguments` is clearly defined. I suggest changing to `((PatternArgs)invocation.Arguments).Args`

Reply:
Changed